clang/lib/CIR/CodeGen/CIRGenExpr.cpp

```diff
@@ -674,6 +727,84 @@ RValue CIRGenFunction::buildLoadOfBitfieldLValue(LValue LV,
   return RValue::get(field);
 }
 
+void CIRGenFunction::buildStoreThroughExtVectorComponentLValue(RValue Src,
+                                                               LValue Dst) {
+  mlir::Location loc = Dst.getExtVectorPointer().getLoc();
```

Comment:
Same for this function, please make sure you add testcases or `llvm_unreachable("NYI");` for paths that are not tested.